schedule/yast/transactional_server/create_hdd_transactional_server.yaml

```diff
@@ -0,0 +1,60 @@
+name:           create_hdd_transactional_server
```

Comment:
add --- here. i am not sure if this wil break the yamllint checks but it is good to follow the specs

Reply:
fixed